pkg/scale/scale.go

```diff
@@ -84,7 +84,16 @@ func (cc *CruiseControlScaler) postCruiseControl(action string, options map[stri
 		return nil, err
 	}
 	if rsp.StatusCode != 200 && rsp.StatusCode != 202 {
-		log.Error(errors.New("Non 200 response from cruise-control: "+rsp.Status), "error during talking to cruise-control")
+		ccErr, parseErr := parseCCErrorFromResp(rsp.Body)
```

Comment:
this was duplicated in multiple cc methods

Reply:
I see the same in line 112, maybe we can change that as well for the sake of completeness.

Reply:
Good catch @hi-im-aren - will change that as well

Reply:
Done - added a similar code in GET method